pkgs/applications/video/dra-cla/default.nix

```
@@ -0,0 +1,43 @@
+{ fetchgit
+, makeWrapper
+, stdenvNoCC
+, lib
+, gnugrep
+, gnused
+, curl
+, mpv
+, aria2
```

The instructions of dra-cla states that ffmpeg is one of it's dependencys 
```suggestion
, aria2
, ffmpeg
```